opm/parser/eclipse/EclipseState/Schedule/RFTConfig.hpp

```diff
@@ -82,27 +81,34 @@ class RFTConfig {
     void addWellOpen(const std::string& well, std::size_t report_step);
 
     const TimeMap& timeMap() const;
-    const std::pair<bool, std::size_t>& wellOpenRftTime() const;
-    const WellOpenTimeMap& wellOpenRftName() const;
-    const WellOpenTimeMap& wellOpen() const;
-    const RFTMap& rftConfig() const;
-    const PLTMap& pltConfig() const;
 
     bool operator==(const RFTConfig& data) const;
 
+    template<class Serializer>
+    void serializeOp(Serializer& serializer)
+    {
+        tm.serializeOp(serializer);
+        serializer(first_rft_event);
+        serializer(well_open_rft_time);
+        serializer(well_open_rft_name);
+        serializer(well_open);
+        serializer.template map<UnorderedMap2,std::string,
+                       DynamicState<std::pair<RFT, std::size_t>>,false>(rft_config);
+        serializer.template map<UnorderedMap2,std::string,
```

Comment:
Wow this looks complicated, Could you explain for mere mortals (small comment)? I guess the UnorderedMap2 type is needed for some special type of serialization because of pair in combination with complex types RFT/PLT.

Reply:
It's simplied now to the extent it should be readable.